local_dispatcher/local_dispatch.py

```diff
@@ -103,6 +104,8 @@ def get_path_with_params(self, path):
         path_with_params = f"{path}?{urllib.parse.unquote(url_parsed.query)}"
         return path_with_params
 
+    def get_request_query_params(self):
+        return dict(urllib.parse.parse_qsl(urllib.parse.urlsplit(request.url).query))
```

Comment:
The previous logic that was used to add query params to the request wasn't encoding query params in the url friendly way so it didn't work with minio.  `get_request_query_params` is used to build a dict out of query params so they can be passed as `params` named argument to `requests.requests` method.